test/wiz/UploadScan.test.js

```
@@ -0,0 +1,61 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const testUtils = require('../utils.js');
+const uploadScan = require('../../src/appmixer/wiz/core/UploadScan/UploadScan.js');
+const moment = require('moment');
```

ESLint

[error] 'moment' is assigned a value but never used.

(no-unused-vars)